### DISCUSSION TITLE
Webpack config question

### DISCUSSION THREAD

**Discussion by jakobnoergaard** [quasarframework/quasar#11030] [General - Components / Directives / etc] [answered]
Hi, and sorry for the newbie question in advance. 

I'm very lost on Webpack and hope someone can help (we are using the new quasar 2.1.5, so it's Webpack 5+). We are using a library that tells us to add the following to our Webpack config:

-----
```js
const webpack = require('webpack');

webpackConfig.resolve = {
  fallback: {
    fs: false,
    crypto: false,
    path: false,

    buffer: require.resolve('buffer/'),
    process: require.resolve('process/browser'),
  },
}

webpackConfig.plugins = [
  ...(webpackConfig.plugins || []),
  new webpack.ProvidePlugin({
      process: 'process/browser',
  }),
]
```
-----

How and where do we do that in the Quasar Config file? I really hope someone can help.

Kind regards,
Jakob

**Comment by jakobnoergaard** [comment, marked as answer]
I fixed it like this:

----
```js
 extendWebpack(cfg, { isServer, isClient }) {
        cfg.resolve.fallback = {
          fs: false,
          crypto: false,
          path: false,
          buffer: require.resolve('buffer/'),
          process: require.resolve('process/browser'),
        }
        cfg.plugins.push(new webpack.ProvidePlugin({ process: 'process/browser' }))
      },
```
---- 

I'm leaving it here for anyone to comment if that was very wrong and everyone else to see if it's correct. 

Feel free to also delete this post if it goes against how things are normally done.

**Reply by metalsadman** [reply]
that's what the `extendWebpack` and `chainWebpack` are for. https://quasar.dev/quasar-cli/handling-webpack#introduction
